src/js/step.js

```diff
@@ -215,12 +215,20 @@ export class Step extends Evented {
    */
   _addKeyDownHandler(element) {
     const KEY_ESC = 27;
+    const LEFT_ARROW = 37;
+    const RIGHT_ARROW = 39;
 
     element.addEventListener('keydown', (e) => {
-      switch(e.keyCode) {
+      switch (e.keyCode) {
```

Comment:
looks like this is the only change from previous PR, I realize you're building on those changes.